packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -678,14 +677,13 @@ class AccountInternal extends PureComponent<
     const account = this.props.accounts.find(
       account => account.id === accountId,
     );
-    return (
-      account &&
-      this.state.search === '' &&
-      this.state.filterConditions.length === 0 &&
-      (this.state.sort === null ||
-        (this.state.sort.field === 'date' &&
-          this.state.sort.ascDesc === 'desc'))
-    );
+    return account
+      ? this.state.search === '' &&
+          this.state.filterConditions.length === 0 &&
+          (this.state.sort === null ||
+            (this.state.sort.field === 'date' &&
+              this.state.sort.ascDesc === 'desc'))
+      : false;
```

Comment:
```suggestion
    if (!account) return false;
    if (this.state.search !== '') return false;
    if (this.state.filterConditions.length > 0) return false;
    
    if (this.state.sort === null) return true;
    else return this.state.sort.field === 'date' &&
              this.state.sort.ascDesc === 'desc';
```

Reply:
Updated as requested (with slight change for linting)